Libraries/Utilities/ErrorUtils.js

```diff
@@ -8,9 +8,7 @@
  *
  * @providesModule ErrorUtils
  */
-/* eslint-disable consistent-this, global-strict */
-
-var GLOBAL = this;
+/* eslint-disable strict */
```

Comment:
We were actually discussing enabling strict mode in modules and maybe remove 'use strict' added manually everywhere so we can leave it as is for now.